Description: add sw64 support
 .
 libbsd (0.11.3-1+0u1) unstable; urgency=medium
 .
   * Add sw64 support
Author: Han Gao <gaohan@uniontech.com>

Last-Update: 2022-04-08

--- libbsd-0.11.3.orig/src/local-elf.h
+++ libbsd-0.11.3/src/local-elf.h
@@ -45,6 +45,12 @@
 #define ELF_TARG_CLASS	ELFCLASS64
 #define ELF_TARG_DATA	ELFDATA2LSB
 
+#elif defined(__sw_64__)
+
+#define ELF_TARG_MACH   EM_SW_64
+#define ELF_TARG_CLASS  ELFCLASS64
+#define ELF_TARG_DATA   ELFDATA2LSB
+
 #elif defined(__amd64__) || defined(__x86_64__)
 
 #define ELF_TARG_MACH	EM_X86_64
